src/sagemaker_pytorch_container/training.py

```diff
@@ -81,12 +81,17 @@ def train(training_environment):
             runner_type = runner.PyTorchXLARunnerType
             logger.info('Invoking PT-XLA Runner')
     logger.info('Invoking user training script.')
+
+    # get capture_error from framework parameters
+    capture_error = training_environment.additional_framework_parameters.get("sagemaker_toolkit_capture_error", True)
```

Comment:
Can we use a different name? `capture_error` is not descriptive and might make user think we will ignore errors when set to False?

Reply:
Updated to `sagemaker_toolkit_native_launcher_enabled`